src/FSharp.Stats.Interactive/test.ipynb

```diff
@@ -0,0 +1,268 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To reproduce the package, run `./Repack.ps1` in powershell. It will clean your cache in `~/.nuget/packages` and pack the library to `FSharp.Stats/pkg` folder, which you should specify below (absolute paths only) in `#i` line.\n",
+    "\n",
+    "The version of the package is always `0.0.0-dev`.\n",
+    "\n",
+    "To rre-run the build/repack you have to restart the notebook kernel, otherwise we cannot remove the referenced local nuget package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "fsharp"
+    },
+    "vscode": {
+     "languageId": "dotnet-interactive.fsharp"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div><div><strong>Restore sources</strong><ul><li><span> C:/Users/schne/source/repos/fslaborg/FSharp.Stats/pkg</span></li></ul></div><div></div><div><strong>Installed Packages</strong><ul><li><span>FSharp.Stats, 0.0.0-dev</span></li><li><span>FSharp.Stats.Interactive, 0.0.0-dev</span></li></ul></div></div>"
```

Comment:
Do you want the cell outputs to be preserved?

Reply:
Yes- i thought it would be nice that you can directly see the output when looking at the sample notebook on github